game/testGame.js

```diff
@@ -237,9 +238,12 @@ describe('test au commencement d\'une partie', function () {
     });
 
     it('cartes visions correctement chargées ', function () {
+
+        let json  = JSON.parse(fs.readFileSync(__dirname + '/cards.json', 'utf8'));
```

Comment:
You can actually require your json file instead of using filesystem.
`const cards = require('./cards.json');`

Reply:
Bad naming here, `json` does not describe what information the variable carries, just its format.